src/Mechanisms/FrontendAssets/FrontendAssets.php

```diff
@@ -163,7 +172,31 @@ public static function js($options)
 
         $token = app()->has('session.store') ? csrf_token() : '';
 
+        $assetWarning = null;
+
         $nonce = isset($options['nonce']) ? "nonce=\"{$options['nonce']}\"" : '';
+        
+        // Use static assets if they have been published
```

Comment:
Can this be extracted into a method or something so it doesn't add too much confusion to this method